### PR TITLE
New package: `@alwatr/icon` Structure

### PR DESCRIPTION
**Pull request by MM25Zamanian**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

Fixes #(issue)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Open an issue for bug report or feature request and mention it to this PR.
- [ ] The commit message follows our guidelines: <https://github.com/AliMD/alwatr/blob/main/CONTRIBUTING.md#commit-message-format>
- [ ] All methods have proper description in `jsdocs` format with `@example` section.
- [ ] Docs have been added / updated (for bug fixes / features).
- [ ] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by MM25Zamanian** [comment]
لطفا سریع تر آیکون پک رو تایین کنید

**Comment by MM25Zamanian** [comment]
```sh
alwatr/
├─ ui/
│  ├─ alwatr-icon/
│  ├─ alwatr-grid/
│  ├─ .../
├─ core/
├─ .../
```
Or
```sh
```

File: packages/ui/icon/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@alwatr/icon",
+  "version": "0.0.1",
+  "description": "Premium icons for Ionic. ",
```
Comment (AliMD) [inline packages/ui/icon/package.json:4]: WTF! ionic?!!

File: packages/ui/icon/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "icon",
+    "component",
+    "typescript",
+    "esm",
+    "alwatr"
+  ],
+  "main": "icon.js",
+  "type": "module",
+  "types": "icon.d.ts",
+  "author": "MohammadMahdi Zamanian <mm25zamanian@gmail.com> (https://mm25zamanian.ir)",
```
Comment (AliMD) [inline packages/ui/icon/package.json:15]: وقتی در یک پروژه مشارک میکنی اسم خودت رو در contributors باید اضافی کنی نه author

File: packages/ui/icon/src/icon.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import {AlwatrElement} from '@alwatr/element';
+import {html, css} from 'lit';
+import {customElement} from 'lit/decorators/custom-element.js';
+
+import type {TemplateResult} from 'lit';
+
+@customElement('alwatr-icon')
+export class AlwatrIcon extends AlwatrElement {
```
Comment (AliMD) [inline packages/ui/icon/src/icon.ts:8]: صحبت های دیشب فراموش شد؟!

File: packages/ui/icon/tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    "composite": true,
+    "tsBuildInfoFile": ".tsbuildinfo",
+    "rootDir": "src",
+    "outDir": "."
+  },
+  // files, include and exclude from the inheriting config are always overwritten.
+  "include": ["src/**/*"],
+  "exclude": [],
+  "references": [
+    // { "path": "../x" },
```
Comment (AliMD) [inline packages/ui/icon/tsconfig.json:13]: element

File: packages/ui/icon/package.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/AliMD/alwatr",
+    "directory": "packages/ui/icon"
+  },
+  "homepage": "https://github.com/AliMD/alwatr/tree/main/packages/ui/icon#readme",
+  "bugs": {
+    "url": "https://github.com/AliMD/alwatr/issues"
+  },
+  "dependencies": {
```
Comment (AliMD) [inline packages/ui/icon/package.json:32]: lit

File: tsconfig.json [hunk 6]
```diff
@@ -113,7 +113,9 @@
     {"path": "./packages/core/nano-server"},
     {"path": "./packages/core/storage"},
     {"path": "./packages/core/token"},
+    {"path": "./packages/core/element"},
```
Comment (AliMD) [inline tsconfig.json:116]: ooops! tnx

**Review by AliMD — CHANGES_REQUESTED** [review]

File: packages/ui/icon/package.json [hunk 7]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (AliMD) [inline packages/ui/icon/package.json:1]: add private

File: packages/ui/icon/src/icon.ts [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+      }
+    `,
+  ];
+
+  @property({reflect: true}) name?: string;
+  @property({reflect: true}) preurl = 'https://cdn.jsdelivr.net/gh/ionic-team/ionicons@6.0.3/src/svg/';
+
+  override render(): TemplateResult {
+    return html`
+      <svg width="512" height="512" viewBox="0 0 512 512">
```
Comment (AliMD) [inline packages/ui/icon/src/icon.ts:34]: replace with new svg

**Review by AliMD — CHANGES_REQUESTED** [review]

**Pull request merged**